docs/restful-api/services/token-services.md

```diff
@@ -228,19 +228,19 @@ Content-Type: application/x-www-form-urlencoded</p></td>
 - <i>DEFAULT</i> - default one-time token (will be expired after first request)<br />
 - <i>SHOP</i> - shop token is used to redirect customer to the NetLicensing Shop ('licenseeNumber' is mandatory)<br />
 - <i>APIKEY</i> - APIKey token<br/>
-- <i>ACTION</i> - Action token (e.g. Customer Portal)<br/>
+- <i>ACTION</i> - Action token (e.g. <a href="customer-portal">Customer Portal</a>)<br/>
```

Comment:
`(e.g. access token for the <a href="customer-portal">Customer Portal</a>)`

Reply:
changed